bioio_ome_zarr/writers/ome_zarr_writer_v3.py

```diff
@@ -0,0 +1,541 @@
+import math
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+
+import numpy as np
+import zarr
+from zarr.codecs import BloscCodec, BloscShuffle
+
+from .axes import Axes
+from .channel import Channel
```

Comment:
any reason not to use the classes from ngff-zarr?

Reply:
Honestly, I didn't look at it. I was having an issue with creating channel info, and creating the object with a to_dict function that could be directly plugged into our metadata was useful. After that was done, I was doing a lot of organizing for metadata for axes and decided to move it into a class as well. These were not supposed to take the place of ngff-zarr classes.